docs/Principles/modeling-principles.md

```diff
@@ -0,0 +1,27 @@
+# Modeling Principles
+
+## Conforms to Schema
+The Monarch Biolink Specification is an implementation of the Biolink Model.
+The KG must be conformant with The Monarch Biolink Specification.
+
+## Node Normalization
+The final KG must have Nodes normalized to the canonical prefix for any given node type.
```

Comment:
Why is Nodes capitalized? If it is to indicate that Nodes is a special term in this context, then you should probably define it. Otherwise, I would assume they're just normal nodes in a (knowledge) graph, which doesn't require a capital N.